lib/bloc/blocs/login_bloc.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:async';
+import 'package:bloc/bloc.dart';
+import 'package:meta/meta.dart';
+import '../login_bloc.dart';
+import '../../repositories/user_repositori.dart';
+
+class LoginBloc extends Bloc<LoginEvent, LoginState> {
+  UserRepository _userRepository;
+
+  LoginBloc({@required UserRepository userRepository})
```

Comment:
```suggestion
  according to Dart's Language tour: 
Flutter instance creation expressions can get complex, so widget constructors use named parameters exclusively. This makes instance creation expressions easier to read.
So if the bloc has simple constructor. we should make LoginBloc({this.userRepository}) for short code and clearly.
```

Reply:
i think we should be add @required anotation in this case and do not using "named constructor" because may be we forget insert "UserRespository" when called in the UI then the application will crash !

Reply:
LoginBloc({this.userRepository}) -> LoginBloc(this.userRepository). should like this.

Reply:
We already convert into getIt so ... =]]]